lang/vi.json

```diff
@@ -3,6 +3,7 @@
     "(and :count more errors)": "(và :count lỗi khác)",
     "A new verification link has been sent to the email address you provided during registration.": "Một liên kết xác minh mới đã được gửi đến địa chỉ email được bạn cung cấp trong quá trình đăng ký.",
```

Comment:
Nên tạo 1 file en.json để lưu message tiếng anh luôn thay vì đặt key dài ntn em nhé
"verificationEmail" : "A new verification link has been sent to the email address you provided during registration"
Rồi ở blade em gọi {{ __("verificationEmail") }} , như thế này thì key sẽ ngắn, gọn với chuẩn hơn
Vì trước đây nó không tìm đc key trong English nên n mới hiển thị cái key đấy lên luôn

Reply:
zạ, có  vài chỗ em thấy không dài lắm thì em khum rút có được khum ạ ?

Reply:
còn vài chỗ dài em rút dòi ạ